pkg/security/resolvers/netns/resolver.go

```diff
@@ -39,10 +39,10 @@ var (
 	// namespace yet.
 	ErrNoNetworkNamespaceHandle = fmt.Errorf("no network namespace handle")
 
-	// lonelyTimeout is the timeout past which a lonely network namespace is expired
-	lonelyTimeout = 30 * time.Second
-	// flushNamespacesPeriod is the period at which the resolver checks if a namespace should be flushed
-	flushNamespacesPeriod = 30 * time.Second
+	// defaultLonelyTimeout is the timeout past which a lonely network namespace is expired
```

Comment:
Based on the comment, I'd change this to something like `defaultLonelyNamespaceTimeout`. But also, what's a lonely network namespace?

Reply:
I changed this to `lonelyNamespaceTimeout` in the last version.
We call a network namespace lonely if it only has local interfaces. Processes in a lonely network namespace cannot communicate outside of their network namespace so we're not interested in their network traffic.